cluster/cluster.go

```diff
@@ -175,30 +174,29 @@ func CreateSCHostService(storageClusterNum string, hostNum string, prefix *strin
 		},
 	}
 
-	res, err := clientset.CoreV1().Services("default").Create(&scSvc);
+	_, err = clientset.CoreV1().Services("default").Create(&scSvc)
 	if err != nil {
-		panic(err.Error())
+		return err
 	}
-	fmt.Println(res.String())
-
+	return nil
 }
 
 //Creates a the "secrets" of a tenant, for now it's just a plain configMap, but it should be upgraded to secret
-func CreateTenantConfigMap(tenantShortName string) {
+func CreateTenantConfigMap(tenant *Tenant) {
```

Comment:
`CreateTenantConfigMap` should return an error instead of panic'ing / printing errors.